scripts/config/templates/nginx.conf.erb

```diff
@@ -359,6 +359,11 @@ http {
     proxy_pass <%= ssr_frontend_host %>/speedy-search$is_args$args;
   }
 
+ # redirect /speedy-search/count and /speedy-search/{listingId} to ssr-frontend
+  location ~ ^/speedy-search/(count|\d+)$ {
+    proxy_pass <%= ssr_frontend_host %>/speedy-search/$1$is_args$args;
```

Comment:
What's considered `$1`? Is it `speedy-search` or `(count|\d+)`?

Reply:
The first regex match, which would be the `(count|\d+)`